src/Microsoft.VisualStudio.Web.CodeGeneration.Msbuild/project.json

```diff
@@ -4,28 +4,29 @@
     "keyFile": "../../tools/Key.snk",
     "warningsAsErrors": true,
     "compile": {
-      "include": "../Shared/**/*.cs"
+      "include": [
+        "../Shared/General/**/*.cs"
+      ]
     },
     "nowarn": [
       "CS1591"
     ],
     "xmlDoc": true
   },
   "dependencies": {
-    "Microsoft.Build": "15.1.319-preview5",
-    "Microsoft.Build.Tasks.Core": "15.1.319-preview5",
-    "Microsoft.Build.Utilities.Core": "15.1.319-preview5",
+    "Microsoft.Build": "14.3.0",
```

Comment:
MSBuild will unify these versions when it loads the task assembly. This should be supported. cref https://github.com/Microsoft/msbuild/issues/1010